src/OrleansProviders/SQLServer/CreateTables.sql

```diff
@@ -1,113 +1,1440 @@
-﻿DROP TABLE [dbo].[OrleansMembershipTable]
-DROP TABLE [dbo].[OrleansMembershipVersionTable]
-DROP TABLE [dbo].[OrleansRemindersTable]
-DROP TABLE [dbo].[OrleansSiloMetricsTable]
-DROP TABLE [dbo].[OrleansClientMetricsTable]
-DROP TABLE [dbo].[OrleansStatisticsTable]
-
-CREATE TABLE [dbo].[OrleansMembershipTable]
+﻿/*
```

Comment:
This, along with ``GenericRelationalStorage`` is the gist of the matter. Most of the design is already commented. A point worth making is that Orleans has a dependency only to the names of the queries and the parameter names and types the queries use. That means, for instance, one can change table name or change functionality so as to, for example, remove ``UPDATE`` clauses as such and insert a new row provided the queries won't see this.

I have a plan to use this also to do queuing etc. Doing it like this doesn't make Orleans dependent on any one vendor and allows for vendor and deployment specific tuning. Also, the only used queries are presented here, there is no need to dig into code.